### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 3

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/algebra#517] [Esercizi M-Z] [answered]
![image](https://github.com/user-attachments/assets/84c3542c-e876-419b-a907-951438bd9c1a)

**Comment by AglaiaNorza** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/b95e926c-7008-49f3-a03e-24bc0290b4a2)

**Comment by erinrrr** [comment]
![image](https://github.com/user-attachments/assets/2330d725-b4db-4f91-a6c3-07321bc887f7)

**Comment by marcoseverini** [comment]
<img width="928" alt="immagine" src="https://github.com/user-attachments/assets/93c58815-5201-4b51-81df-35ebb3be770c">
